SHIR/setup.ps1

```diff
@@ -135,7 +135,7 @@ Set-Environment-Varibles-From-Secrets
 # Register SHIR with key from Env Variable: AUTH_KEY
 if (Check-Is-Registered) {
     Write-Log "Restart the existing node"
-    Start-Process $DmgcmdPath -Wait -ArgumentList "-Start"
+    Start-Process $DmgcmdPath -Wait -ArgumentList "-Start -TurnOffAutoUpdate"
```

Comment:
Turning off AutoUpdate as its not a supported feature of Windows containers SHIR per https://docs.microsoft.com/en-us/azure/data-factory/how-to-run-self-hosted-integration-runtime-in-windows-container#limitations

Instead the scheduled run will update the base image with the latest SHIR version and the flux will redeploy it.